java-ast/tests/pom.xml

```diff
@@ -41,6 +41,12 @@
             <artifactId>windup-exec</artifactId>
             <classifier>forge-addon</classifier>
             <scope>test</scope>
+        </dependency>    
+        <dependency>
+            <groupId>org.eclipse.tycho</groupId>
+            <artifactId>org.eclipse.jdt.core</artifactId>
+            <version>3.10.0.v20140604-1726</version>
+            <scope>test</scope>
```

Comment:
BTW @mareknovotny my OS locale is in spanish (LANG=es_ES.UTF-8). Didn't know this can have impact in tests. Additionally, I don't think this is related to the first test failure. Can you try testing in a empty environment? (maybe remove all in your local .m2 repo and clone windup master again).